src/tools/index.md

```diff
@@ -96,7 +96,7 @@ The Dart SDK includes the following general-purpose tools:
   testing, compiling, and running Dart code,
   as well as working with the [pub package manager](/guides/packages).
 
-[`dartdoc`](/tools/dartdoc)
+[`dartdoc`](/tools/dart-doc)
```

Comment:
Do you want to change these and similar mentions to `dart doc` as well?

```suggestion
[`dart doc`](/tools/dart-doc)
```

Reply:
I suspect that we want to change every single mention of `dartdoc` to `dart doc`, but also (in the canonical discussions) mention that it used to be called `dartdoc`.